src/replication.c

```diff
@@ -2867,14 +2864,8 @@ static void dualChannelFullSyncWithPrimary(connection *conn) {
         connClose(server.repl_transfer_s);
         server.repl_transfer_s = NULL;
     }
-    if (server.repl_rdb_transfer_s) {
```

Comment:
remove this since replicationAbortDualChannelSyncTransfer will handle it as well